### PR TITLE
fix: update kms key policy, sns policy, stackset instance creation

### PR DESCRIPTION
**Pull request by ipcrm**
Additional updates required for member accounts to use SNS topic, add dependencies so stack can be torn down, and to create stackset instances so deployment occurs.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -258,11 +280,13 @@ data "aws_iam_policy_document" "sns_topic_policy" {
...
   statement {
-    actions = ["sns:Publish"]
+    actions = [
+      "sns:Publish",
+    ]
 
     condition {
       test     = "StringEquals"
-      variable = "aws:PrincipleOrgID"
+      variable = "aws:PrincipalOrgID"
```
Comment (afiune) [inline main.tf:289]: really!

File: main.tf [hunk 2]
```diff
@@ -321,5 +353,25 @@ resource "aws_cloudformation_stack_set" "lacework_stackset" {
...
+    aws_secretsmanager_secret.lacework_api_credentials,
+    aws_lambda_function.lacework_setup_function
+  ]
+}
+
+
+data "aws_region" "current" {}
+resource "aws_cloudformation_stack_set_instance" "lacework_stackset_instances" {
+  deployment_targets {
+    organizational_unit_ids = [var.organization_unit]
```
Comment (afiune) [inline main.tf:372]: Is this only 1 org unit? or can this be multiple ones?

**Review by afiune — DISMISSED** [review]

File: main.tf (hunk 2 above)
Comment (ipcrm) [reply main.tf:372]: Can be multiple! I've left like this ATM to match expectations of the generate command. If we want to make this an array (totally fine) we need to update generate. 

Can do now or raise a separate issue.

**Pull request merged**